vignettes/explanation/node-js-javascript-and-sass-tools.Rmd

```diff
@@ -2,35 +2,78 @@
 title: "Explanation: Node.js - JavaScript and Sass tools"
 output: rmarkdown::html_vignette
 vignette: >
+  %\VignetteEncoding{UTF-8}
   %\VignetteIndexEntry{Explanation: Node.js - JavaScript and Sass tools}
   %\VignetteEngine{knitr::rmarkdown}
-  %\VignetteEncoding{UTF-8}
+editor_options: 
+  markdown: 
+    wrap: 72
```

Comment:
Do we need this? Also, we wrap around column 100 in general.

Reply:
This was generated automatically by my RStudio options, I usually use wrapping around 80. I will remove those lines

Reply:
Fixed